plugins/GSdx/GSState.h

```diff
@@ -243,6 +243,7 @@ class GSState : public GSAlignedClass<32>
 	void ResetHandlers();
 
 	int GetFramebufferHeight();
+	void SaturateOutputSize(GSVector4i*);
```

Comment:
Returning the value sounds like a good idea.

> As far as I can see, I only see a demerit that the function call wouldn't have & which was necessary to show that the variable's value might be modified by the function.

For pointer and reference parameters, the presence or absence of `const` is a better indicator of whether a value might be modified. References/pointers are also used in cases where passing by value is inefficient.